### PR TITLE
PR for lm-logs-release-0.3.3

### PR DESCRIPTION
**Pull request by choudharypooja**
PR for ticket https://jira.logicmonitor.com/browse/DEV-111918

### CODE REVIEW

**Review by shubhashri12 — APPROVED** [review]

**Review by ashishkucheria — APPROVED** [review]

File: lm-logs/values.yaml [hunk 1]
```diff
@@ -24,6 +24,9 @@ fluent:
       chunk_limit_size: 8m
       flush_thread_count: 8
 
+kubernetes:
+  multiline_start_regexp: /^\[(\d{4}-)?\d{2}-\d{2} \d{2}:\d{2}:\d{2}.\d{3}.*\]/
```
Comment (ahsan13jan) [inline lm-logs/values.yaml:28]: We should not keep a default, It is upon customer to customer what they want to add it as.

File: lm-logs/templates/configmap.yaml [hunk 2]
```diff
@@ -65,7 +66,21 @@ data:
         </pattern>
       </parse>
     </source>
-    
+
```
Comment (ahsan13jan) [inline lm-logs/templates/configmap.yaml:69]: What is the customer does want this multiline start regex, what if it wants use the default `\n`?

**Review by ahsan13jan — CHANGES_REQUESTED** [review]
We should not default it to use the multiline logs plugin, if the customer wants it they can add it,

**Review by ahsan13jan — APPROVED** [review]

**Review by vkumbhar94 — CHANGES_REQUESTED** [review]
seems you have built chart before bumping up the version, hence docs/lm-logs-0.3.2.tgz is showing in diff

**Review by siddharthck — APPROVED** [review]

**Review by vkumbhar94 — APPROVED** [review]

**Pull request merged**